components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -344,6 +349,18 @@ void BravePrefProvider::UpdateCookieRules(ContentSettingsType content_type,
     rules.emplace_back(CloneRule(firebase_rule));
     brave_cookie_rules_[incognito].emplace_back(CloneRule(firebase_rule));
   }
+
+  if (prefs_->GetBoolean(kHangoutsEnabled)) {
```

Comment:
can we please extract this and the exceptions above into a separate function? Do stop code bloat here at this method (also cc @bridiver )